### ISSUE TITLE
[Docos] Unbearable typo in "Type Narrowing" FAQ entry :zombie: 

### ISSUE THREAD

**Issue by antonioan** [beartype/beartype#467]
Thanks for the awesome work! In the following you meant to munch on list of _integers_, not strings:

```py
    # If this list contains integers rather than strings, call another
    # function accepting only a list of integers.
    if is_bearable(lst, list[int]):
        # "lst" has been though a lot. Let's celebrate its courageous story.
        munch_on_list_of_strings(lst)  # mypy/pyright: OK!
```
Source: https://beartype.readthedocs.io/en/latest/faq/#with-type-narrowing

**Comment by leycec** [comment]
**OMG!** The horrifying visage of this documentation bug is truly despicable. Thanks for the rapid heads-up, @antonioan. Let's resolve this for both your sanity and mine. :smile:

**Comment by leycec** [comment]
**Resolved by c3339a397ba.** I... I did something on a Sunday. My wife and I can't believe it. Truly, Christmas miracles do exist. :laughing:
